src/main/kotlin/mods/eln/mechanical/RadialMotor.kt

```diff
@@ -70,18 +71,31 @@ class RadialMotorDescriptor(baseName: String, obj: Obj3D) :
     override val obj: Obj3D = obj
 
     override fun addInformation(stack: ItemStack, player: EntityPlayer, list: MutableList<String>, par4: Boolean) {
-        list.add("Converts $fluidDescription into mechanical energy.")
-        list.add("Nominal usage ->")
-        list.add("  ${fluidDescription.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() }} input: $fluidConsumption mB/s")
+//        list.add("Converts $fluidDescription into mechanical energy.")
+//        list.add("Nominal usage ->")
+//        list.add("  ${fluidDescription.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() }} input: $fluidConsumption mB/s")
+//        if (power.isEmpty()) {
+//            list.add("  No valid fluids for this turbine!")
+//        } else if (power.size == 1) {
+//            list.add(Utils.plotPower("  Power out: ", power[0]))
+//        } else {
+//            list.add("  Power out: ${Utils.plotPower(minFluidPower  * GAS_GUZZLER_CONSTANT)}- ${Utils.plotPower(maxFluidPower * GAS_GUZZLER_CONSTANT)}")
+//        }
+//        list.add(Utils.plotRads("  Optimal rads: ", optimalRads))
+//        list.add(Utils.plotRads("Max rads:  ", absoluteMaximumShaftSpeed))
+
```

Comment:
I would avoid adding commented lines here. Can we delete them if the new code works?

Reply:
These commented lines are my code before the changes I made. I commented them to ensure that I could restore them in case the modifications failed. However, I forgot to remove them after completing the changes.You can delete it directly.